vault/changelog.md

```diff
@@ -2,11 +2,27 @@
 id: 9bc92432-a24c-492b-b831-4d5378c1692b
 title: Changelog
 desc: ''
-updated: 1658249180078
+updated: 1658759707340
 created: 1601508213606
 nav_order: 2
 ---
 
+## 0.105.0
+
+### Enhancements
+
+- enhance(views): option to filter hierarchical edges from graph panel (#3243) @joshi ([[docs|dendron://dendron.dendron-site/dendron.topic.sidebar.graph-panel#toggle-edges]])
+- enhance(publish): use note's `desc`field for SEO description (#3261) @sam ([[docs|dendron://dendron.dendron-site/dendron.topic.publish.config.seo.description#^ygk9kha1hgzy]])
+- enhance(retrieve): support end block anchor (#3248) @kevin ([[docs|dendron://dendron.dendron-site/dendron.topic.note-reference#end-positional-reference]])
```

Comment:
one more change to add to both the enhancements as well as to the release notes:

- enhance(preview): open note in different editor group then preview (#3278) @kevin